_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -53,45 +44,41 @@ Some user agents treat the value of `aria-*` attribute as case-sensitive (even w
 - [WAI-ARIA 1.1, Characteristics of States and Properties, Value](https://www.w3.org/TR/wai-aria/#propcharacteristic_value)
 - [Uniform Resource Identifier (URI): Generic Syntax (RFC 3986)](https://www.ietf.org/rfc/rfc3986.txt)
 
+Depending on the role of an elements, certain WAI-ARIA properties are or not required. This rule considers that when such a property has a value type of `ID reference` or `ID reference list`, then an element with one of the given ids must exist. On another hand, if a property that is not required refers to an incorrect id, it is less clear that this will cause an accessibility issue (the property is not required on the first place, so the element can live without it, and maybe later interaction with the page will create the correct id and this is the intended behavior). Therefore, the rule has no extra requirement for properties with type `ID reference` or `ID reference list` that are not required.
+
+For value type `URI`, this rule does not require that the destination URI exists. Testing that the link is not dead is out of scope for the rule and therefore ignored.
+
 ## Test Cases
 
 ### Passed
 
 #### Passed Example 1
 
-`aria-required` property with valid true/false value
+This `div` element has an `aria-required` property with valid `true/false` value.
 
 ```html
 <div role="textbox" aria-required="true" aria-label="A required textbox"></div>
```

Comment:
Why are we using divs for this at all? Couldn't we just use native elements here? I think a mix of native and ARIA role elements would be better.